frontend/src/GlobalStyle.tsx

```diff
@@ -59,5 +59,10 @@ const GLOBAL_STYLE = css({
     "body": {
         fontFamily: "'Open Sans', sans-serif",
         fontWeight: 400,
+
+        // From a set of popular phones, the iPhone 5 has the smallest viewport
+        // width: 320px. It does make sense to set a minimum width early on in
+        // order to know where we can stop caring.
+        minWidth: 320,
```

Comment:
Normally I would grumble here, but even Chrome on a 1080p desktop doesn't let me zoom further in than to whatever level makes the viewport `381.03px` wide. :sweat_smile:

Reply:
Yeah, 320px is really not a lot. And I am pretty sure that for people who need to zoom that far, it's better to scroll horizontally a bit instead of the website making crude adjustments (probably hiding content) to fit into <320px.